src/site/content/en/blog/speed-to-business/index.md

```diff
@@ -0,0 +1,177 @@
+
+**Slow down the page**
+
+This may be the easiest way to create variants and can be achieved by adding a simple script, slowing down server response times, loading larger images, etc. The Financial Times opted for this option when testing how performance impacted their business metrics: see [A faster FT.com](https://medium.com/ft-product-technology/a-faster-ft-com-10e7c077dc1c).
+
+**Speed up page load**
+
+For cases where the test page (let's say a product detail page) is mostly linked out from a different page (let's say the homepage), [prefetching](/link-prefetch/) or [prerendering](https://developers.google.com/web/updates/2018/07/nostate-prefetch) the product page directly from the homepage for the test group will speed up the subsequent load of the page. Note that in this case the A/B test split (step 4) is done on the homepage. In addition, all of this may slow down the first page so be sure to measure that and take it under consideration when analyzing the test results (step 5).
+
+
+## Step 4: Create an A/B test
+
+Once you have two versions of the same page where one is faster than the other, the next step is to split traffic to measure impact. In general there are many techniques and tools to perform A/B tests, but be aware that not all methods are well suited to measure speed performance impact.
+
+If you're using an A/B testing tool like [Optimizely](https://www.optimizely.com/de/) or [Optimize](https://optimize.google.com/optimize/home/) we strongly recommend setting up a server-side test instead of a client-side test, as client side A/B testing often works by hiding page content until the experiment is loaded, which would mean the A/B testing itself would skew the metrics you wanted to measure. If you can only do client-side testing, consider setting up the experiment on a different page and changing the link outs to your test page to split the traffic. This way the test page itself isn't dragged down by the client-side test.
+
+{% Details %}
+
+{% DetailsSummary %}
+
+Example
+
+{% endDetailsSummary %}
+
+Example of AB testing  performance changes on a given product detail page (PDP) via server-side testing:
+
+![Server side testing diagram](test-server-side.jpg)
+
+The request goes to the backend, which distributes the users to the two different versions of the page. While this is in general a good setup, it often needs IT resources to set up the server-side split.
+
+Here is an example of a client side testing setup, using the prior page (the homepage in the diagram below) to run the testing JavaScript:
+
+![Client side testing diagram](test-client-side.jpg)
+
+The testing JavaScript manipulates the outgoing link to give the two test groups of users links to the two versions of the PDP in question. This is easy to setup and maintain through common AB testing tools like Optimizely or Optimize and does not influence the performance test because the testing JavaScript runs on a different page.
```

Comment:
```suggestion
The testing JavaScript manipulates the outgoing link to give the two test groups of users links to the two versions of the PDP in question. This is easy to setup and maintain through common A/B testing tools like Optimizely or Optimize and does not influence the performance test because the testing JavaScript runs on a different page.
```